profiles/tests/test_profiles_graphql_gdpr_api.py

```diff
@@ -500,7 +500,7 @@ def test_user_tries_deleting_his_profile_but_it_fails_partially(
     ServiceConnectionFactory(profile=profile, service=service_2)
 
     def get_response(request, context):
-        if not request.body or "dry_run" not in request.body:
+        if "dry_run=true" not in request.url:
```

Comment:
The `request` object has both `qs` and `query` attributes which could be more appropriate for this check. Applies also to a few more similar places in this PR.